CONTRIBUTING.md

```diff
@@ -59,6 +59,12 @@ Changes should conform to:
 
 With the exception that 4-space indendation be used.
 
+#### General guidelines for Swift for TensorFlow API docs contribution
+
+* Closely mirror Swift for TensorFlow API docs with implementations in [tensorflow/tensorflow/python](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/python) to maintain consistency across platforms.
+  - If required, replace variable names from Python docs to reflect the Swift code. For example: `y_true` and `y_predict` in TensorFlow 2.x API docs will become `expected` and `predicted` in Swift for TensorFlow, respectively.
+* When you contribute a new feature to Swift for TensorFlow, the maintenance burden is (by default) transferred to the Swift to TensorFlow team. This means that the benefit of the contribution must be compared against the cost of maintaining the feature.
```

Comment:
I think this bullet can be removed. It's not related to API documentation, and the points above about testing already convey a similar message:

```
#### Testing

*   Include unit tests when you contribute new features, as they help to a)
    prove that your code works correctly, and b) guard against future breaking
    changes to lower the maintenance cost.
*   Bug fixes also generally require unit tests, because the presence of bugs
    usually indicates insufficient test coverage.
```

Reply:
Resolving via https://github.com/tensorflow/swift-apis/pull/589#discussion_r360610805.